bitcash/base58.py

```diff
@@ -49,19 +49,20 @@ def b58decode(string):
             num *= 58
             num += alphabet_index[char]
     except KeyError:
-        raise ValueError('"{}" is an invalid base58 encoded '
-                         'character.'.format(char)) from None
+        raise ValueError(
+            '"{}" is an invalid base58 encoded ' "character.".format(char)
```

Comment:
I'm not thrilled with this change. I need to look more closely.

Reply:
I submitted this: https://github.com/psf/black/issues/1338

Not sure if this is okay to clean up one time. `black`'s master branch has the same issue.

Will wait at least a couple days to hear back from the developers and see.

I guess this is not as straight forward as I thought. I was hoping it would be more bulletproof like `go fmt`.